src/charm.py

```diff
@@ -118,6 +132,47 @@ def __init__(self, *args):
             self._certificates.on.certificate_expiring, self._on_certificate_expiring
         )
 
+    def _configure_amf(self, _ : EventBase) -> None:
+        """Handle Juju events.
+
+        Whenever a Juju event is emitted, this method performs a couple of checks to make sure that
+        the workload is ready to be started. Then, it configures the AMF workload,
+        runs the Pebble services and expose the service information through charm's interface.
```

Comment:
This makes it sound as if this code is triggered for every Juju event which is not quite the case. 

We may want to include the following:
```
This event handler is called for every event that affects the charm state (ex. configuration files, relation data)
```

Reply:
Done